account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -0,0 +1,130 @@
+import {
+  custom,
+  keccak256,
+  parseEther,
+  publicActions,
+  toHex,
```

Comment:
⚠️ **[eslint]** <[@typescript-eslint/no-unused-vars](https://typescript-eslint.io/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'toHex' is defined but never used.
<!-- __reviewdog__:ChBiMGQ5Nzg4MGI0NjU5YTM2EgZyZGpzb24= -->